pkg/parser/parser_test.go

```diff
@@ -1,4 +1,4 @@
-package parser_test
+package parser
```

Comment:
Renamed the package name so that I can use the constants defined in package `parse`

Reply:
changed package name back to parser_test as it's ginkgo's convention